tools/sim/tests/test_sim_bridge.py

```diff
@@ -62,17 +62,27 @@ def test_engage(self):
 
     while time.monotonic() < start_time + max_time_per_step:
       sm.update()
-
       q.put("cruise_down")  # Try engaging
-
       if sm.all_alive() and sm['controlsState'].active:
         control_active += 1
-
         if control_active == min_counts_control_active:
           break
-
     self.assertEqual(min_counts_control_active, control_active, f"Simulator did not engage a minimal of {min_counts_control_active} steps was {control_active}")
 
+    # Drive course
+    q.put("reset")
+    start_time = time.monotonic()
+    user_disengage_once = False
+    disengage_events = ('stockAeb', 'fcw', 'ldw')
```

Comment:
this doesn't tell you what you need to know. the point of the bounty is to determine whether openpilot stayed in its lane, drove a reasonable, etc. and finished the course